### DISCUSSION TITLE
I want to immigrate version to get resolved issues that we are facing because of lower version.

### DISCUSSION THREAD

**Discussion by jadhavarjun** [UI5/webcomponents-react#6332] [Q&A]
currently in my package.json we have using below mentioned versions. 
  _"@ui5/webcomponents": "^1.14.0",
    "@ui5/webcomponents-base": "^1.14.0",
    "@ui5/webcomponents-fiori": "^1.14.0",
    "@ui5/webcomponents-icons": "^1.14.0",
    "@ui5/webcomponents-react": "^1.16.3",
    "@ui5/webcomponents-react-base": "^1.16.3",_

and I want to upgrade the version _"@ui5/webcomponents-react": "2.2.0",  "@ui5/webcomponents-react-base": "2.2.0"",_ so i wanted to know while upgrading @ui5/webcomponents-react": "2.2.0" which version I should need to upgrade in other libraries and which other libraries also need to upgrade. and while migrating from 1.14.0 to 2.2.0  version which type of issue will be occurred and how should I have resolved them and which things I keep in mind while immigrating versions.

NOTE: It is urgent matter.

**Comment by jadhavarjun** [comment]
@Lukas742, @ui5-webcomponents-react-bot

**Comment by MarcusNotheis** [comment]
Closing as duplicate of https://github.com/SAP/ui5-webcomponents-react/issues/6326.
